bowling/DirectoryContent.hpp

```diff
@@ -0,0 +1,23 @@
+#include <algorithm>
+#include <filesystem>
+#include <iostream>
```

Comment:
why u add this in this file?

Reply:
> why u add this in this file?

Filesystem is for path object - to iterate through folder